xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3011,6 +3011,7 @@ platform.livetable.asyncActionInProgress=In progress...
 platform.livetable.asyncActionDone=Done
 platform.livetable.asyncActionFailed=Failed
 platform.livetable.filtersTitle=Filter for the {0} column
+platform.livetable.filter.date.format.description=date format: {0} - {0}
```

Comment:
Is it usual to have such a short description? I would have rather expected something like "Date range filter, enter the date range in the format {0} or use the date range picker.". Also, I would suggest to pass the whole date range format as a single parameter instead of having `{0} - {0}` in the translation as this will confuse translators and translators might mess up the format.